src/py42/clients/settings/device_settings.py

```diff
@@ -0,0 +1,349 @@
+from py42._compat import str
+from py42._compat import UserDict
+from py42._compat import UserList
+from py42.clients.settings import SettingProperty
+from py42.clients.settings import show_change
+from py42.clients.settings._converters import bool_to_str
+from py42.clients.settings._converters import days_to_minutes
+from py42.clients.settings._converters import minutes_to_days
+from py42.clients.settings._converters import str_to_bool
+from py42.exceptions import Py42Error
+
+
+class DeviceSettingsDefaults(UserDict, object):
+    """Helper class for managing Org Device Default settings. Also acts as a base class
+    for `DeviceSettings` to manage individual device settings."""
+
+    def __init__(self, device_dict, org_settings):
+        self.data = device_dict
+        self._org_settings = org_settings
+        self.changes = org_settings.changes
+        self._destinations = org_settings.data["settings"]["destinations"]
+        self.data["settings"] = {
+            "serviceBackupConfig": self.data["serviceBackupConfig"]
+        }
+        self.backup_sets = [
+            BackupSet(self, self.changes, backup_set)
+            for backup_set in self.data["serviceBackupConfig"]["backupConfig"][
+                "backupSets"
+            ]
+        ]
+
+    @property
+    def available_destinations(self):
+        return {d["guid"]: d["destinationName"] for d in self._destinations}
+
+    warning_email_enabled = SettingProperty(
+        name="warning_email_enabled",
+        location=["settings", "serviceBackupConfig", "warningEmailEnabled"],
+        get_converter=str_to_bool,
+        set_converter=bool_to_str,
+    )
+    critical_email_enabled = SettingProperty(
+        name="critical_email_enabled",
+        location=["settings", "serviceBackupConfig", "severeEmailEnabled"],
+        get_converter=str_to_bool,
+        set_converter=bool_to_str,
+    )
+    warning_alert_days = SettingProperty(
+        name="warning_alert_days",
+        location=["settings", "serviceBackupConfig", "minutesUntilWarning"],
+        get_converter=minutes_to_days,
+        set_converter=days_to_minutes,
+    )
+    critical_alert_days = SettingProperty(
```

Comment:
It must `critical` in the UI I am guessing? But `severe` on the API

Reply:
Yep, there are a lot of mismatches between what the endpoint expects and what the UI calls it, so we're always mimicking the UI.